src/wasm/wasm-emscripten.cpp

```diff
@@ -361,29 +358,28 @@ inline void exportFunction(Module& wasm, Name name, bool must_export) {
   wasm.addExport(exp);
 }
 
-void EmscriptenGlueGenerator::generateDynCallThunk(std::string sig) {
-  auto* funcType = ensureFunctionType(sig, &wasm);
+void EmscriptenGlueGenerator::generateDynCallThunk(Signature sig) {
   if (!sigs.insert(sig).second) {
     return; // sig is already in the set
   }
-  Name name = std::string("dynCall_") + sig;
+  Name name = std::string("dynCall_") + "TODO: generate sig string";
```

Comment:
What is the status of this TODO?

Reply:
Embarrassingly skipped. Thankfully it's a trivial fix.